### PR TITLE
NXP-25146: add responsive action menus to web ui

### PR DESCRIPTION
**Pull request by Gabez0r**
Depends on nuxeo/nuxeo-ui-elements#276.

### CODE REVIEW

**Comment by guirenard** [comment]
:+1:

File: elements/diff/nuxeo-document-diff-button.html [hunk 1]
```diff
@@ -64,6 +75,10 @@
         this.fire('nuxeo-diff-documents', {
           documents: this.selectedDocuments
         });
+      },
+
+      _computeLabel: function() {
```
Comment (nmpcunha) [inline elements/diff/nuxeo-document-diff-button.html]: Should you use i18n argument?

**Comment by nmpcunha** [comment]
👍

File: elements/diff/nuxeo-document-diff-button.html (hunk 1 above)
Comment (Gabez0r) [reply elements/diff/nuxeo-document-diff-button.html]: It's just a trigger.

**Pull request merged**